genTestPKI.go

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
Also, if these are fixtures for the http transport testing then this go:generate should be in the transport/http package.

Reply:
this means one will have to run `go generate` in transport/http package.

with the current change, we can run `go generate` from the root of the project and the artifacts will be generated in /test folder

Reply:
The generate goes with the code that needs it, not at the base of the project.

Reply:
file removed, generate is now under the http transport package